packages/next-swc/crates/next-core/js/src/ipc/index.ts

```diff
@@ -109,6 +109,13 @@ function createIpc<TIncoming, TOutgoing>(
     })
   }
 
+  // When the socket is closed, this process is no longer needed.
+  // This might happen e. g. when parent process is killed or
+  // node.js pool is garbage collected.
+  socket.once('close', () => {
+    process.exit(0)
```

Comment:
Isn't this done just above?
- first: https://github.com/vercel/next.js/blob/bbe82117ff6c9432c1cf26dad16c9da0e1e649ed/packages/next-swc/crates/next-core/js/src/ipc/index.ts#L86-L91
- duplicate: https://github.com/vercel/next.js/blob/bbe82117ff6c9432c1cf26dad16c9da0e1e649ed/packages/next-swc/crates/next-core/js/src/ipc/index.ts#L112-L117

Reply:
oh oops, that slipped in from random local changes